### DISCUSSION TITLE
Japanese Localization

### DISCUSSION THREAD

**Discussion by mhara** [Sequel-Ace/Sequel-Ace#1386] [General] [answered]
Hello. I am Makoto Hara, Japanese translator of Sequel Pro. (was almost 10 years ago :-)
May I comment/review on the current Japanese translations in Crowdin?
Because it is almost done, but has quite a few mistranslations.
Thank you.

**Comment by Kaspik** [comment, marked as answer]
You can go ahead and update them / fix them :)
